internal/ocistorage/ociwrapper.go

```diff
@@ -280,9 +280,19 @@ func (o *OCIWrapper) DeleteManifest(ctx context.Context, opts OCIOpts, id string
 	return repo.Delete(ctx, desc)
 }
 
-// ExperimentalOCIIsEnabled returns true if the EXPERIMENTAL_OCI_STORAGE env variable is set to true
+// OCIIsEnabled returns true if the OCI_STORAGE env variable is not set to true
```

Comment:
Unless I've missed an important aspect of this:
```suggestion
// OCIIsEnabled returns true if the OCI_STORAGE env variable is not set to false
```

Reply:
I changed it to reflect that it's `true` also when it's not set